### DISCUSSION TITLE
Is possible use with `@nuxt/http ` ?

### DISCUSSION THREAD

**Discussion by chuoke** [nuxt-community/auth-module#1479] [Ideas & Feature Requests]
I find 
> we are using @nuxt/http which we recommend for fetching data from an API.

**Comment by sausin** [comment]
The code seems to us `axios` for requests, but I would also be very interested in possibility of using with `@nuxt/http`

**Comment by bmulholland** [comment]
If this is still something people want: what do you mean "use with?" What doesn't already work? What should work differently?
